railties/lib/rails/commands/rake/rake_command.rb

```diff
@@ -19,6 +19,9 @@ def perform(task, args, config)
             load "rails/tasks.rb"
             rake.init("rails", [task, *args])
             rake.load_rakefile
+            if Rails.respond_to?(:root)
+              rake.options.suppress_backtrace_pattern = /\A(?!#{Regexp.quote(Rails.root.to_s)})/
```

Comment:
So if you use `--trace` this would still show the full backtrace right?

Reply:
Correct!